### DISCUSSION TITLE
Time string not always shown

### DISCUSSION THREAD

**Discussion by sylr** [go-delve/delve#3339] [Q&A] [answered]
Any idea why sometimes Time string is not displayed in VSCode ?

<img width="787" alt="image" src="https://user-images.githubusercontent.com/153052/234211247-38113fdd-848f-40bb-aa56-01209f1d806a.png">

**Comment by aarzilli** [comment, marked as answer]
I think it's a bug, file an issue with more details.

**Reply by sylr** [reply]
Done https://github.com/go-delve/delve/issues/3342, thank you.
